assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-title-block/edit.tsx

```diff
@@ -1,23 +1,34 @@
 /**
  * External dependencies
  */
-import { useBlockProps } from '@wordpress/block-editor';
-
-/**
- * Internal dependencies
- */
-import Block from './block';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export const Edit = (): JSX.Element => {
 	const blockProps = useBlockProps();
```

Comment:
Not a big deal, so feel free to ignore this. But IMO it would be a good idea to make sure the same class names from the frontend are applied in the editor.
```suggestion
	const blockProps = useBlockProps( {
		className: 'wc-block-mini-cart__title',
	} );
```
This would allow that if a theme adds some CSS styles to these elements, they are visible in the editor as well. It would also allow us to remove some styles from `assets/js/blocks/mini-cart/mini-cart-contents/editor.scss` as they will be inherited from `assets/js/blocks/mini-cart/style.scss`. (It will also require some other CSS resets, but I think it's worth it)

Reply:
👍